src/plays/drumkit/styles.css

```diff
@@ -0,0 +1,82 @@
+@import url('https://fonts.googleapis.com/css2?family=Arvo:wght@100;200;300;400;500;600;700&display=swap');
+
+.play-details-body {
+  text-align: center;
+  background-color: #283149;
+}
+
+h1 {
+  font-size: 5rem;
+  color: #dbedf3;
+  font-family: 'Arvo', cursive;
+  text-shadow: 3px 0 #da0463;
+}
+

```

Comment:
please do not target directly `h1`. add a separate name. You can see if we target `h1` directly it will change the other things too.

### Preview deployment
![image](https://github.com/reactplay/react-play/assets/88102392/62c81eca-ed14-4717-bfd5-252f78dfd5e8)

![image](https://github.com/reactplay/react-play/assets/88102392/0bb34854-e950-403d-98a7-cbdad038608b)
### Production site 
![image](https://github.com/reactplay/react-play/assets/88102392/fba44212-4e3f-4e97-af30-1fe6400a553c)

Reply:
@priyankarpal sure, will do that.